src/driver/tests/experimental_trial_test.py

```diff
@@ -432,7 +432,7 @@ def test_evaluate_single_publisher_model(self):
             # been checked by the preceding unit tests.  However, we make a few strategic
             # probes.
             self.assertEqual(result.shape[0], 1)
-            self.assertAlmostEqual(result["relative_error_at_100"][0], 0.0, delta=0.01)
+            self.assertAlmostEqual(result["relative_error_at_100"][0], 0.0, delta=0.02)
```

Comment:
I don't know about the computation behind "relative_error_at_100". Could you confirm that the increase of `delta` is due to computational error but not some logical error?

Reply:
Yes I indeed should check it. 

1. In the main branch with original codes, I varied the `seed` in `trial.evaluate` above.  For `seed in range(200`, 6 seeds lead to error > 0.01 (and <0.02).  So, it is a computational error and delta=0.02 is reasonable.
2. This test does not use Copula generator at all, so is not affected by this PR except for the randomness.

PLMK what you think.

Reply:
Thanks for checking. That sounds good to me.